test/functional/api/deprecated_spec.lua

```diff
@@ -18,4 +18,19 @@ describe('deprecated', function()
       n.api.nvim_notify('hello world', 4, {})
     end)
   end)
+
+  describe('nvim_*get_option functions', function()
+    it('does not leak memory', function()
+      if not (n.is_asan() or os.getenv('VALGRIND')) then
+        pending('not running in valgrind or asan')
+        return
+      end
```

Comment:
I don't think this is needed. A test is usually only skipped if it cannot be run.

Reply:
Removed.
But just curious, doesn't this mean a test "pass" would give misleading false negative if it breaks again?

Reply:
Does that matter? Usually one only looks at a test if it fails.